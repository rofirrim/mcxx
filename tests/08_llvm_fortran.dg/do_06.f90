! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="do_06.f90.out"
! </testinfo>
SUBROUTINE SUB(S)
    INTEGER :: I, S
    DO I = 5, 1, S
        PRINT *, I
    END DO
END SUBROUTINE SUB

PROGRAM MAIN
    CALL SUB(-1)
    CALL SUB(-2)
END PROGRAM MAIN
