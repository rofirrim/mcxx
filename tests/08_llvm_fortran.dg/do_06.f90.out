           5
           4
           3
           2
           1
           5
           3
           1
